### PR TITLE
Fix responsive dos and dont's by change min width and fix tipseen story

### PR DESCRIPTION
**Pull request by hadasfa**
Change min width of do/don't rule in stories
override default for tipseen (the examples are bigger there

### CODE REVIEW

File: src/components/Tipseen/__stories__/tipseen.stories.mdx [hunk 1]
```diff
@@ -145,74 +157,80 @@ Use when the tip is too long or when you want to teach something in steps.
...
-      )
+      );
     }}
   </Story>
 </Canvas>
 
 ## Do’s and Don’ts
+
 <ComponentRules
+  className="monday-storybook-tipseen_rules"
```
Comment (hadasfa) [inline src/components/Tipseen/__stories__/tipseen.stories.mdx:175]: the other things in this file are lint related, this is the only real change

File: src/components/Tipseen/__stories__/tipseen.stories.scss [hunk 2]
```diff
@@ -1,7 +1,19 @@
 @import "../../../styles/themes";
+@import "../../../styles/global-css-settings";
 @import "../../../styles/typography";
 
+$component-rules-grid-gap: $spacing-xxl;
```
Comment (hadasfa) [inline src/components/Tipseen/__stories__/tipseen.stories.scss:5]: it's not beautiful I know :(

**Comment by hadasfa** [comment]
It's not a beautiful solution but it is working and I did not want to refactor the mixin and damage it for give an override option which right now needed only for one use case (I thought it will be overkill)

File: src/components/Tipseen/__stories__/tipseen.stories.scss (hunk 2 above)
Comment (SergeyRoyt) [reply src/components/Tipseen/__stories__/tipseen.stories.scss:5]: why u need individual variables for that?

**Review by SergeyRoyt — APPROVED** [review]

**Pull request merged**